go-controller/pkg/node/bridgeconfig/bridgeconfig_testutil.go

```
@@ -0,0 +1,137 @@
+package bridgeconfig
+
+import (
+	"fmt"
+	utilnet "k8s.io/utils/net"
```

golangci-lint (1.64.8)

File is not properly formatted

(gci)